common/src/main/java/com/wynntils/features/alwayson/FixCastingSpellsFromInventoryFeature.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.alwayson;
+
+import com.wynntils.core.features.StateManagedFeature;
+import com.wynntils.mc.event.ArmSwingEvent;
+import net.minecraft.world.InteractionHand;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+/**
+ * This feature fixes an issue where a Archers holding bows may start
+ * casting spells just by dropping items from the inventory screen.
+ */
+public class FixCastingSpellsFromInventoryFeature extends StateManagedFeature {
+
+    @SubscribeEvent
+    public void onSetSlot(ArmSwingEvent event) {
+        if (event.getActionContext() == ArmSwingEvent.ArmSwingContext.DROP_ITEM_FROM_INVENTORY_SCREEN
```

Comment:
This might have gotten lost in our conversation (or I failed to communicate this), but we don't really need all of the mixins to post events for all action contexts when we only use one? I think they should he removed, and only post when context is this one (and we can just extend the event when we need it). Keeping the enum is fine.